tools/fiptool/fiptool.c

```diff
@@ -528,7 +528,7 @@ static int pack_images(const char *filename, uint64_t toc_flags, unsigned long a
 
 	/* Append a null uuid entry to mark the end of ToC entries. */
 	memset(toc_entry, 0, sizeof(*toc_entry));
-	toc_entry->offset_address = entry_offset;
+	toc_entry->offset_address = (entry_offset + align - 1) & ~(align - 1);
```

Comment:
It's better to add some comments here, the last toc_entry offset address is the value of fip size. It will make sure the next change will not break the rule.
Thanks.